### PR TITLE
minor script for testing

### PR DESCRIPTION
**Pull request by kipiiler**

### CODE REVIEW

**Comment by elimelt** [comment]
probably better to rewrite these to call the admin commands

File: scripts/windows/addAllToPool.bat [hunk 1]
```diff
@@ -0,0 +1,3 @@
+set "SQL_QUERY= insert into interview_interviewpool select id from members_user"
+
+docker exec -it swecc-server-db-1 psql -U root postgres -c "%SQL_QUERY%"
```
Comment (elimelt) [inline scripts/windows/addAllToPool.bat:3]: can do this without sql so it also works in prod (not that we should ever run this in prod)

```bash
container_id=$1
cmd="put_all_users_in_pool"
docker exec -it $container_id python server/manage.py $cmd
```

or w/e it is.

File: scripts/windows/listUser.bat [hunk 2]
```diff
@@ -0,0 +1,3 @@
+set "SQL_QUERY= select id, username from members_user"
```
Comment (elimelt) [inline scripts/windows/listUser.bat:1]: same here. we also have a list users admin command

**Review by elimelt — CHANGES_REQUESTED** [review]
you should use django management commands for all scripting since it lets you interact with our domain objects instead of writing raw sql, and is cross platform by default. I've already written both of these as python management commands, but if you want to add a windows stub to all of these to make it easier to trigger then feel free

**Comment by ShawnCollinge** [comment]
@elimelt we can close this since your admin dashboard does this right?

**Pull request closed without merging**